docs/user-guide/installation/fedora.rst

```diff
@@ -20,10 +20,15 @@ MongoDB replica sets if you'd like to have higher availability.
 
    $ sudo dnf install mongodb-server
 
-You need mongodb-server with version >= 2.4 installed for Pulp server. It is highly recommended
-that you `configure MongoDB to use SSL`_. If you are using Mongo's authorization feature, you
-will need to grant the ``readWrite`` and ``dbAdmin`` roles to the user you provision for Pulp to
-use. The ``dbAdmin`` role allows Pulp to create collections and install indices on them.
+You need mongodb-server with version >= 2.4 installed for Pulp server. MongoDB 2.x reached its EOL.
```

Comment:
Would it be possible to require mongodb 3.x?

Reply:
https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#python-driver-compatibility

and pymongo as well :(

Reply:
Cool, I have [a task open](https://pulp.plan.io/issues/3259) where we can maybe figure out how to drop support for MongoDB 2. We can continue the conversation there.